### ISSUE TITLE
no-callback-literal

### ISSUE THREAD

**Issue by Friph** [ricardofbarros/linter-js-standard#211]
Hello,

Could you tell me where is the problem? It's on every first line:
2 Examples:
![bildschirmfoto 2017-11-10 um 11 13 57](https://user-images.githubusercontent.com/31510303/32653460-459a4104-c608-11e7-9edf-6dada945411c.png)
![bildschirmfoto 2017-11-10 um 11 14 43](https://user-images.githubusercontent.com/31510303/32653492-62e4d3a0-c608-11e7-9058-560cd7a40916.png)

And the linter error: 

![bildschirmfoto 2017-11-10 um 11 15 22](https://user-images.githubusercontent.com/31510303/32653518-77712c2e-c608-11e7-8fb1-91909e26450e.png)

Thanks!

**Comment by sonicdoe** [comment]
Do you have `standard` locally installed? Does this error also occur when running `standard` from the command line?

**Comment by Friph** [comment]
Yes the error also occurred with the local installation in `node_modules/.bin/standard` running from the command line.

The issue was apparently that we had an old version of standard installed, because yarn did not correctly update the dependencies. A related issue is https://github.com/standard/snazzy/issues/16

Thanks for the fast response and the heads up and sorry for the delayed answer.

**TL;DR;**
We `rm -rf node_modules/` and reinstall with `yarn install` or `npm install`.
